use_cases/MODELS_INDEX.md

```diff
@@ -0,0 +1,18 @@
+| Use Case              | XML File Path                                                                     | Location       | Variables              | \# Data Years | \# Years in Output | AR order | MA order | Fourier Periods                                                                       | Pivot Length | \# Clusters | Preserve Input CDF? |
```

Comment:
Is it possible to move pivotLengths and # Clusters to after the `Variables` column? In my mind those columns group better with the numerical input specs.